torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1850,6 +1850,85 @@ def sample_inputs_stack(op_info, device, dtype, requires_grad, **kwargs):
         for dim in range(-1, len(shape) - 1):
             yield SampleInput(tensors, args=(dim,))
 
+def error_inputs_cat(op_info, device, **kwargs):
+    # error inputs for unsupported operation
```

Comment:
Why is it unsupported?

Reply:
It's because more than one tensor cannot refer to the same memory locations. Added the comment there! :) Thanks